### DISCUSSION TITLE
G7 sensor age discrepencies

### DISCUSSION THREAD

**Discussion by jgslade** [NightscoutFoundation/xDrip#3249] [Bug reports] [answered]
The Classic Status Page is showing the start date of my previous sensor but the  G5/G6/G7 status page and the home screen are showing the correct sensor age.

![image](https://github.com/NightscoutFoundation/xDrip/assets/29897815/4e3be447-e37f-4c3e-b756-2a80dc44eb05)

![image](https://github.com/NightscoutFoundation/xDrip/assets/29897815/713bf680-d20f-42a6-8b51-9cc781a3bb9e)

![image](https://github.com/NightscoutFoundation/xDrip/assets/29897815/38ebcc52-03bc-418e-b39c-74afed068881)

**Comment by Navid200** [comment]
We don't need to start sensor for G7.
A lot you see on the classic status page are invalid for G7 and are what worked for G6.
I have an open submission that will change G6 native to G7 after it is approved.

Bluetooth device and connection status are invalid because G7 transmitter ID is not related to the bluetooth name of the device.  As a result, xDrip cannot show those two and it also cannot forget device.  If you want to forget a device, you will have to use Android settings.

I have another submission that removes Bluetooth device, connection status and forget device from that page.
We may figure out a workaround for any of those later and bring them back.

The sensor start time is as you said the last time you started a sensor.

**Reply by jgslade** [reply]
My issue is the start date on the Classic page is what is now being sent to AAPS. Xdrip knows the correct start info for the G7, but the data being sent to AAPS isn't that data. The data being sent is the data on the Classic page.

**Reply by Navid200** [reply, marked as answer]
Please click on create issue from discussion in the right pane.

For the title of the issue that you will be opening, I suggest something like:
Invalid G7 start time sent to AAPS

My point is that whoever is going to fix this needs to be someone who has experience with AAPS.  Leaving that out of the title will not be good for that reason.
Thanks
